README.md

```diff
@@ -419,6 +419,22 @@ LLDPA configuration file: /etc/lldap/lldap_config.toml<br>
 </table>
 LLDPA configuration file: /etc/lldap/lldap_config.toml<br>
 </details>
+<details>
+<summary><b>FreeBSD</b></summary>
+<br>
+  Official FreeBSD support is offered through the <a href="https://www.freshports.org/">openSUSE Build Service</a>.<br><br>
+  Maintainer: <a href="https://github.com/aokblast">@aokblast</a><br>
+  Support: <a href="https://bugs.freebsd.org/bugzilla/">Bugzilla</a>, <a href="https://github.com/lldap/lldap/discussions">Discussions</a><br>
+  Package repository: <a href="https://www.freshports.org/net/lldap/">FreeBSD Freshport Build</a><br>
+<table>
+  <tr>
+    <td>Available packages:</td>
+    <td>lldap</td>
+    <td>Light LDAP server for authentication.</td>
+  </tr>
+</table>
+LLDPA configuration file: /usr/local/lldap_server/lldap_config.toml<br>
```

Comment:
LLDAP :)
That's copy paste, but I think the other instances were fixed in the meantime, if you want to rebase.